src/key.h

```diff
@@ -17,7 +17,7 @@
 
 
 /**
- * secure_allocator is defined in allocators.h
+ * secure_allocator is defined in allocators/secure.h
```

Comment:
Good catch.
I'd, personally, prefer removing this line completely. If you want to find out where it is defined, you can find it.

Reply:
Fixed